submit-header-design.md

```diff
@@ -126,10 +126,10 @@ Once you have an idea for your theme and want to make it real, then the best way
    label = "View Figma file"
 %}
 
-Once your design is in good shape and you'd like to get some first feedback from the Bitcoin Design Community, you have two options. You can either post a link or screenshot with some background information to the #design-review channel in our Slack workspace. Or you can [create a Github issue](https://github.com/BitcoinDesign/Guide/issues){:target="_blank"}. Or you can do both, as you prefer.
+Once your design is in good shape and you'd like to get some first feedback from the Bitcoin Design Community, you have two options. You can either post a link or screenshot with some background information to the #design-crits channel in our Discord workspace. Or you can [create a Github issue](https://github.com/BitcoinDesign/Guide/issues){:target="_blank"}. Or you can do both, as you prefer.
```

Comment:
I see a channel called #design-reviews but not #design-crits

Reply:
Fixed